Remove building of docker images.

---
diff --git a/cmd/server/build.sh b/cmd/server/build.sh
index 1c540285c..e62d49432 100755
--- a/cmd/server/build.sh
+++ b/cmd/server/build.sh
@@ -99,10 +92,3 @@ IMAGE=sourcegraph/grafana:server CACHE=true docker-images/grafana/build.sh

 echo "--- jaeger-all-in-one binary"
 cmd/server/jaeger.sh
-
-echo "--- docker build"
-docker build -f cmd/server/Dockerfile -t "$IMAGE" "$OUTPUT" \
-  --progress=plain \
-  --build-arg COMMIT_SHA \
-  --build-arg DATE \
-  --build-arg VERSION
diff --git a/docker-images/prometheus/build.sh b/docker-images/prometheus/build.sh
index 944594d46..f66fec9bd 100755
--- a/docker-images/prometheus/build.sh
+++ b/docker-images/prometheus/build.sh
@@ -47,12 +43,5 @@ if [[ "$CACHE" == "true" ]]; then
   BUILD_CACHE=""
 fi
 
-# shellcheck disable=SC2086
-docker build ${BUILD_CACHE} -t "${IMAGE:-sourcegraph/prometheus}" . \
-  --progress=plain \
-  --build-arg COMMIT_SHA \
-  --build-arg DATE \
-  --build-arg VERSION
-
 # cd out of $BUILDDIR for cleanup
 popd
