diff --git a/cmd/server/build.sh b/cmd/server/build.sh
index e62d49432..f47054f4b 100755
--- a/cmd/server/build.sh
+++ b/cmd/server/build.sh
@@ -88,7 +88,6 @@ IMAGE=sourcegraph/prometheus:server CACHE=true docker-images/prometheus/build.sh
 echo "--- grafana"
 cp -r docker-images/grafana/config "$OUTPUT/sg_config_grafana"
 cp -r dev/grafana/linux "$OUTPUT/sg_config_grafana/provisioning/datasources"
-IMAGE=sourcegraph/grafana:server CACHE=true docker-images/grafana/build.sh
 
 echo "--- jaeger-all-in-one binary"
 cmd/server/jaeger.sh
